README.md

```diff
@@ -1,48 +1,58 @@
-In addition EclipseSource also offers [professional support](https://jsonforms.io/support) for JSON Forms.
+## Feedback, Help and Support
+
+Feel free to start a discussion by [opening an issue](https://github.com/eclipsesource/jsonforms/issues/new/choose) on the repo or by contacting us [directly via email](mailto:jsonforms@eclipsesource.com?subject=JSON%20Forms).
+You can also use the [discussion board](https://spectrum.chat/jsonforms) to get help from the community.
+In addition, EclipseSource also offers [professional support](https://jsonforms.io/support) for JSON Forms.
 
-# Developers Documentation
+## Developers Documentation
+
+### First time setup
 
-## First time setup
 * Install [node.js](https://nodejs.org/) (version >= 6.x.x)
 * Update npm (version >= 5.8.0)
 * Clone this repository
 * Install dependencies: `npm ci`
 * Hook up dependencies between packages: `npm run init`
 
-## Build & Testing
+### Build & Testing
+
 * Build (all packages): `npm run build`
 * Test (all packages): `npm run test`
 * Clean (delete `dist` folder of all packages): `npm run clean`
 * Run vanilla examples: `cd packages/vanilla && npm run dev`
 * Run material examples: `cd packages/material && npm run dev`
-* Check Formatting: `npm run check-format`
 
-## Dependency & Release management
-For more info about how we handle dependencies and releases in the JSON Forms project, please see our [Developer Documentation wiki page](https://github.com/eclipsesource/jsonforms/wiki/Developer-documentation). 
+### Dependency & Release management
 
-## Continuous Integration
-The JSONForms project is build and tested via [Travis](https://travis-ci.org/). Coverage is documented by [Coveralls](https://coveralls.io).
+For more info about how we handle dependencies and releases in the JSON Forms project, please see our [Developer Documentation wiki page](https://github.com/eclipsesource/jsonforms/wiki/Developer-documentation).
+
+### Continuous Integration
+
+The JSON Forms project is build and tested via [Travis](https://travis-ci.org/). Coverage is documented by [Coveralls](https://coveralls.io).
 
 Current status: [![Build Status](https://travis-ci.org/eclipsesource/jsonforms.svg?branch=master)](https://travis-ci.org/eclipsesource/jsonforms) [![Coverage Status](https://coveralls.io/repos/eclipsesource/jsonforms/badge.svg?branch=master&service=github)](https://coveralls.io/github/eclipsesource/jsonforms?branch=master)
 
-# License
-The JSONForms project is licensed under the MIT License. See the [LICENSE file](https://github.com/eclipsesource/jsonforms/blob/master/LICENSE) for more information.
+## License
+
+The JSON Forms project is licensed under the MIT License. See the [LICENSE file](https://github.com/eclipsesource/jsonforms/blob/master/LICENSE) for more information.
+
+## Migration
+
+If you are still using JSON Forms 1, check our [migration guide](https://github.com/eclipsesource/jsonforms/blob/master/MIGRATION.md).
```

Comment:
The migration guide also contains information about other versions, reformulate to something like this: `See our migration guide when updating JSON Forms`.